Sources/Widgets/Widgets3D/ShapeWidget/behavior.js

```diff
@@ -371,10 +371,7 @@ export default function widgetBehavior(publicAPI, model) {
     ) {
       return macro.VOID;
     }
-
     if (!model.point2) {
-      // Update orientation to match the camera's plane
-      // if the corners are not yet placed
```

Comment:
Can you restore those comments (unless they're inaccurate)? All those widgets are pretty confusing already, so I'd rather you add more if possible 😉 Thanks.